.gitignore

```diff
@@ -6,3 +6,4 @@ dist
 *.swp
 doc/_build
 *.egg-info
+.idea
```

Comment:
What's that? Never heard of it

Reply:
jetbrains IDEs create this folder.
This just prevents from adding it accidently.